config/initializers/wicked_pdf.rb

```diff
@@ -0,0 +1,18 @@
+# WickedPDF Global Configuration
+#
+# Use this to set up shared configuration options for your entire application.
+# Any of the configuration options shown here can also be applied to single
+# models by passing arguments to the `render :pdf` call.
+#
+# To learn more, check out the README:
+#
+# https://github.com/mileszs/wicked_pdf/blob/master/README.md
+
+WickedPdf.config ||= {}
```

Comment:
why do you use `||=`? 🤔

Reply:
Honestly I followed the guide for setup and working with wicked_pdf where such settings were advised.
[This is link to guide](https://blog.corsego.com/gem-wicked-pdf#:~:text=(If%20it%20is%20set%20up%20this%20way%2C%20it%20will%20work%20correctly%20on%20heroku)%20config/initializers/wicked_pdf.rb)